src/test/scala/edu/ie3/simona/test/common/input/PvInputTestData.scala

```diff
@@ -55,6 +58,34 @@ trait PvInputTestData
     0.95,
   )
 
+  private val operationTimeBuilder = OperationTime.builder()
+
+  private val interval = new ClosedInterval[ZonedDateTime](
+    TimeUtil.withDefaults.toZonedDateTime("2020-01-01T11:00:00Z"),
+    TimeUtil.withDefaults.toZonedDateTime("2020-01-01T14:00:00Z"),
+  )
+  private val operationTime: OperationTime =
+    operationTimeBuilder.withOperationTime(interval).build()
+
+  protected val pvInputWithQCharacteristicLimitedOperationTime = new PvInput(
+    UUID.randomUUID(),
+    "Dummy_PvModel_With_Q_Characteristic",
+    new OperatorInput(UUID.randomUUID(), "NO_OPERATOR"),
+    operationTime,
+    nodeInputNoSlackNs04KvA,
+    new CosPhiFixed("cosPhiFixed:{(0.0,0.95)}"),
```

Comment:
Hmm, I think there's even an argument for changing the `pvInput` to not having a cos phi of 1. But that's entirely optional...

Reply:
I would like to have the reactive power of Pv included since we check also (a tiny bit) that we have a pv with capacitve character.

Reply:
To that point as well, all the former PvInput test data has mismatching cosPhiRated of 0.95 and qCharacteristics of fixed 1